controllers/lmes/lmevaljob_controller.go

```diff
@@ -942,6 +942,10 @@ func CreatePod(svcOpts *serviceOptions, job *lmesv1alpha1.LMEvalJob, log logr.Lo
 			Name:  "UNITXT_ALLOW_UNVERIFIED_CODE",
 			Value: "True",
 		},
+		{
+			Name:  "HF_ALLOW_CODE_EVAL",
+			Value: "1",
+		},
```

Comment:
If this is dependent on the allowRemote field, should we explicitly add

```go
	{
			Name:  "HF_ALLOW_CODE_EVAL",
			Value: "0",
		},
```

to `disallowRemoteCodeEnvVars`?